docs/developer/serving/services/byo-certificate.md

```diff
@@ -0,0 +1,65 @@
+# Custom TLS certificate for DomainMapping
+
```

Comment:
Should we add `{{ feature(beta="0.24") }}` like [Configuring custom domains section](https://github.com/knative/docs/blob/mkdocs/docs/developer/serving/services/custom-domains.md)?